### PR TITLE
Setup as package

### PR DESCRIPTION
**Pull request by DanielAndreasen**
Following this guide: https://python-packaging.readthedocs.io/en/latest/minimal.html

1. Should be able to install a new package `FASMA` with `pip install -e .`
2. Can run tests with the command `nosetests` from rootdir (have to run `pip install -r test_rquirements.txt` first)

### CODE REVIEW

File: FASMA/interpolation.py [hunk 1]
```diff
@@ -262,22 +261,20 @@ def interpolator(
     params = list(params)
     if atmtype == 'marcs':
         newatm = interpolator_marcs(params, fesun=7.47, microlim=3.0)
-    elif atmtype == 'kurucz95':
-        newatm = interpolator_kurucz(params, atmtype=atmtype)
     elif atmtype == 'apogee_kurucz':
         newatm = interpolator_kurucz(params, atmtype=atmtype)
     else:
-        raise NameError(f'Could not find {atmtype} models')
+        raise NameError('Could not find %s models', atmtype)
```
Comment (DanielAndreasen) [inline FASMA/interpolation.py]: Why this change? Not a fan of f-strings?

Comment (MariaTsantaki) [reply FASMA/interpolation.py]: The problem is that f-string require python=>3.6. They don't work for 3.5.

Comment (DanielAndreasen) [reply FASMA/interpolation.py]: Oh. That's a shame. It's so elegant.

Comment (MariaTsantaki) [reply FASMA/interpolation.py]: I agree

Comment (DanielAndreasen) [reply FASMA/interpolation.py]: Do you think we can merge this branch now? Then we can move on with the tests

Comment (MariaTsantaki) [reply FASMA/interpolation.py]: Yes. let's merge
